tcms/telemetry/templates/telemetry/testing/test-case-health.html

```diff
@@ -14,6 +14,7 @@ <h2>{% trans "Most frequently failing test cases" %}</h2>
             <thead>
                 <tr>
                     <th class="header"> {% trans "Test Case" %}</th>
+                    <th class="header" style="width: 100px"> </th>
```

Comment:
try min-width or something similar b/c when screen is resized this column gets resized as well and the progress bar squares become aligned vertically.

Reply:
Done. I also reduced that to 70px, since 100px is too big for 5 boxes of 7px.